src/components/scenes/newAccount/NewAccountReviewScene.tsx

```diff
@@ -116,18 +113,15 @@ interface UpgradeReviewProps extends SceneProps<'upgradeAccountReview'> {
 }
 
 export const UpgradeReviewScene = (props: UpgradeReviewProps) => {
-  const { branding, route } = props
-  const dispatch = useDispatch()
+  const { route } = props
   const { onComplete, onLogEvent = (event, values?) => {} } = useImports()
+  const dispatch = useDispatch()
 
   const handleNext = useHandler(() => {
     onLogEvent(`Backup_Review_Done`)
 
     if (onComplete != null) onComplete()
-
-    dispatch(checkAndRequestNotifications(branding)).catch(error =>
-      console.log(error)
-    )
+    dispatch(completeLogin(route.params.account))
```

Comment:
I don't think this makes sense - the upgrade flow is for an already logged-in account, which means they would have already seen the modal when they logged in a few seconds ago.